CONTRIBUTING.md

```diff
@@ -6,5 +6,15 @@ Please be aware, BeeWare operates under a Code of Conduct.
 
 See [CONTRIBUTING to BeeWare](http://beeware.org/contributing) for details.
 
-See also the Briefcase specific contributing guides for contributing [code](https://briefcase.readthedocs.io/en/latest/how-to/contribute-code.html) and [documentation](https://briefcase.readthedocs.io/en/latest/how-to/contribute-docs.html)
+See also the Briefcase specific contributing guides for contributing
+[code](https://briefcase.readthedocs.io/en/latest/how-to/contribute-code.html)
+and
+[documentation](https://briefcase.readthedocs.io/en/latest/how-to/contribute-docs.html)
 
+## Development Environment
+
+Run `pip install pre-commit` to install pre-commit hooks in to your git hooks.
```

Comment:
What happens if a user *doesn't* do this? (Because you can guarantee 90% of first time contributors won't, even if it *is* in the documentation...)